_dev/src/utils/SSCFilters.ts

```diff
@@ -137,3 +141,14 @@ export function findAndCheckFilter(
   checkAndUpdateDimensionStatus(dimension);
   return dimension;
 }
+
+export function retrieveProductNumberFromFiltersIds(
+  productFilters :filtersChosenFromAPI[], dimension:DimensionChosen[],
```

Comment:
```suggestion
  productFilters: filtersChosenFromAPI[], dimensions: DimensionChosen[],
```

As it is an array, it is better to write it in plural form